src/lib/components/intl-tel-input.component.ts

```diff
@@ -9,9 +9,7 @@
 
 import { AfterViewInit, Component, ElementRef, EventEmitter, Input, Output, ViewChild } from '@angular/core';
 import { ControlContainer, NgForm } from '@angular/forms';
-import intlTelInput from 'intl-tel-input';
-import { IntlTelInputOptions } from '../model/intl-tel-input-options';
-import { IntlTelInput } from "../model/intl-tel-input";
+import intlTelInput, {Iti, SomeOptions} from 'intl-tel-input';
```

Comment:
tsconfig.json needs to be changed, like this
```json
    "paths": {
      "intl-tel-input": [
        "node_modules/intl-tel-input/build/js/intlTelInput.d.ts"
      ]
    },
```

Reply:
I think this is an error from upstream, will open a PR for discussion. We shouldn't need that

Reply:
https://github.com/jackocnr/intl-tel-input/pull/1561

Reply:
21.0.8 fixes this, so not needed. You may `npm update` anyway, and see how it goes.